CounterStrike/src/dev/danablend/counterstrike/listeners/PlayerDropItemListener.java

```diff
@@ -24,11 +24,10 @@ public void playerItemDropEvent(PlayerDropItemEvent event) {
         if (!event.getPlayer().isOp()) {
             int currentSlot = event.getPlayer().getInventory().getHeldItemSlot();
 
-            if (currentSlot == 0 || currentSlot == 1) {
-                return;
+            //can't drop knife or shop if active
+            if (currentSlot == 2 || currentSlot == 8) {
+                event.setCancelled(true);

```

Comment:
In order to increase readability, there might be final static integers created, like `public final int GRENADE_SLOT = 1;`

Then instead of using numbers in comparisons, meaningful names like GRENADE_SLOT or KNIFE_SLOT could be used.